CIP-onchain-cert-metadata/README.md

```diff
@@ -0,0 +1,489 @@
+---
+CIP: ?
+Title: On-chain Certification Metadata Standard
```

Comment:
```suggestion
Title: On-chain dApp Certification Metadata Standard
```
Since this proposal is aimed at dApp certification, perhaps changing to the title to reflect this may be fitting.

Reply:
@rphair has a good eye for naming :)

Reply:
@Ryun1 you are absolutely right: this text of this proposal is 100% related to dApps at this time.  @RSoulatIOHK I would also propose that we remove "Standard" from the CIP title, which is implied and sometimes unwanted (since by design CIPs always permit alternatives).  Will retitle it here & in PR title to avoid clutter at the top of the manuscript & hope in advance it's OK with the authors. 🙏